app/curses_util.py

```diff
@@ -75,36 +91,52 @@
   KEY_CTRL_UP = 562
   KEY_CTRL_SHIFT_UP = 563
 else:
-  KEY_ALT_LEFT = 542
-  KEY_ALT_RIGHT = 557
-  KEY_ALT_SHIFT_LEFT = 543
-  KEY_ALT_SHIFT_RIGHT = 558
-  KEY_CTRL_DOWN = 524
-  KEY_CTRL_SHIFT_DOWN = 525
-  KEY_CTRL_LEFT = 544
-  KEY_CTRL_SHIFT_LEFT = 545
-  KEY_CTRL_RIGHT = 559
-  KEY_CTRL_SHIFT_RIGHT = 560
-  KEY_CTRL_UP = 565
-  KEY_CTRL_SHIFT_UP = 566
+  KEY_ALT_LEFT = 'kLFT3'
+  KEY_ALT_RIGHT = 'kRIT3'
+  KEY_ALT_SHIFT_LEFT = 'kLFT4'
+  KEY_ALT_SHIFT_RIGHT = 'kRIT4'
+  KEY_CTRL_DOWN = 'kDN5'
+  KEY_CTRL_SHIFT_DOWN = 'kDN6'
+  KEY_CTRL_LEFT = 'kLFT5'
```

Comment:
Keen, on darwin ctrl+left sends 541 and on Linux it's 544, but both convert to kLFT5 -- which may be exactly what that string label is trying to help with (make it so we don't have to workout all the codes). Guessing, but seems that seems reasonable.